rst/conf.py

```diff
@@ -47,8 +46,8 @@
 nitpicky = True
 
 # General information about the project.
-project = u'Python-LLFUSE'
-copyright = u'2010-2015, Nikolaus Rath'
+project = 'Python-LLFUSE'
+copyright = '2010-2015, Nikolaus Rath'
```

Comment:
guess this depends on how the docs are built and might need changes on the system doing that.

Reply:
@Nikratio can you comment on this?

Reply:
As long as no one tries to build the documentation with Python 2.x, that should be fine.

Reply:
ok, so i assume this is not the case for llfuse docs.